README.md

```diff
@@ -1,60 +1,39 @@
-## Task Description
+# City temperature mangement
 
-You are required to create a FastAPI application that manages city data and their corresponding temperature data. The application will have two main components (apps):
+Project for managing cities(CRUD), getting and updating temperature records for every city in DB
 
-1. A CRUD (Create, Read, Update, Delete) API for managing city data.
-2. An API that fetches current temperature data for all cities in the database and stores this data in the database. This API should also provide a list endpoint to retrieve the history of all temperature data.
 
-### Part 1: City CRUD API
+## Installation
+- Python 3.7+ must be installed (3.9+ prefer)
+- Clone this repo on your PC
+- Sign up on https://www.weatherapi.com/ and get API_KEY
 
-1. Create a new FastAPI application.
-2. Define a Pydantic model `City` with the following fields:
-    - `id`: a unique identifier for the city.
-    - `name`: the name of the city.
-    - `additional_info`: any additional information about the city.
-3. Implement a SQLite database using SQLAlchemy and create a corresponding `City` table.
-4. Implement the following endpoints:
-    - `POST /cities`: Create a new city.
-    - `GET /cities`: Get a list of all cities.
-    - **Optional**: `GET /cities/{city_id}`: Get the details of a specific city.
-    - **Optional**: `PUT /cities/{city_id}`: Update the details of a specific city.
-    - `DELETE /cities/{city_id}`: Delete a specific city.
+```shell
+python -m venv venv # Create virtual environment
+venv\Scripts\activate # Activate venv for Windows
+venv/bin/activate # Activate venv for MacOS
+pip install -r requirements.txt # Install needed dependencies
+```
+Create .env file in project root directory with variables from .env.sample
+```shell
+alembic init alembic # Initiate alembic
+alembic revision --autogenerate -m "Name commit" # Input name you want
```

Comment:
If you already have the necessary migrations in your project and you're confident that they are correctly set up, you may not need it